### PR TITLE
style(*): Propagate Jboss Logmanager

### PR DESCRIPTION
**Pull request by aoudiamoncef**

### CODE REVIEW

File: core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java [hunk 1]
```diff
@@ -29,7 +28,7 @@
 
     @Substitute
     public static Logger getLogger(Class<?> clazz) {
-        return LoggerFactory.getLogger(clazz.getName());
+        return Logger.getLogger(clazz.getName());
```
Comment (dmlloyd) [inline core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java]: This is definitely incorrect.  The substitution is meant to return an SLF4J logger, not a JBoss logger.

**Review by dmlloyd — DISMISSED** [review]

File: core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java (hunk 1 above)
Comment (Sparow199) [reply core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java]: Oops, i'll fix that.

Comment (Sparow199) [reply core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java]: @dmlloyd , what do you think about choosing single Logger name convention in Quarkus ?

```java
private static final Logger LOGGER logger  LOG  log // which form do you suggest ?
```

Comment (dmlloyd) [reply core/runtime/src/main/java/io/quarkus/runtime/graal/LoggingSubstitutions.java]: I don't have a very strong opinion, but I use `log` or `LOG` myself.

**Comment by Sparow199** [comment]
@dmlloyd ,
What do you think if we choose a unified Logger name convention ? i found:
 ```private static final LOGGER, logger, LOG, log``` 
and different kind of instantiations:
 ```Logger.getLogger with Test.class, Test.class.getName(), "Test"```

**Comment by dmlloyd** [comment]
I don't like using the class name as the log category.  But I also don't like arguing about it very much!  I would recommend using a category name that groups together messages that relate to whatever process is being executed using `Logger.getLogger(String)`.  If people insist on using the class name, then `Logger.getLogger(Class<?>)` is the appropriate way to do it.

**Comment by Sparow199** [comment]
Thanks for your feedback. Do you think that failing tests are caused by the changes in this PR ?

**Comment by machi1990** [comment]
> Thanks for your feedback. Do you think that failing tests are caused by the changes in this PR ?

I am seeing this is error 
```
Caused by: java.lang.ClassFormatError: Duplicate method name "findById" with signature "(Ljava.lang.Object;)Ljava.lang.Object;" in class file io/quarkus/it/mongodb/panache/bugs/Bug5274EntityRepository
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at io.quarkus.it.mongodb.panache.bugs.Bug5274EntityRepository_Bean.<init>(Bug5274EntityRepository_Bean.zig:143)
	at io.quarkus.arc.setup.Default_ComponentsProvider.addBeans1(Default_ComponentsProvider.zig:227)
	at io.quarkus.arc.setup.Default_ComponentsProvider.getComponents(Default_ComponentsProvider.zig:38)
	at io.quarkus.arc.impl.ArcContainerImpl.<init>(ArcContainerImpl.java:102)
	at io.quarkus.arc.Arc.initialize(Arc.java:19)
	at io.quarkus.arc.runtime.ArcRecorder.getContainer(ArcRecorder.java:34)
	at io.quarkus.deployment.steps.ArcProcessor$generateResources15.deploy_0(ArcProcessor$generateResources15.zig:72)
	at io.quarkus.deployment.steps.ArcProcessor$generateResources15.deploy(ArcProcessor$generateResources15.zig:36)
	at io.quarkus.runner.ApplicationImpl.<clinit>(ApplicationImpl.zig:296)
```

I remember @FroMage fixed it recently here https://github.com/quarkusio/quarkus/pull/5276.

**Comment by FroMage** [comment]
WTH? This test passed when we merged it, no?

**Comment by machi1990** [comment]
> WTH? This test passed when we merged it, no?

Yeah, CI was happy when it was merged.

**Comment by FroMage** [comment]
Just tried it on master and it passes.

**Comment by FroMage** [comment]
Perhaps caching went wrong and it was using an older version of mongo-panache, let's restart the test.

**Comment by FroMage** [comment]
Single job restart still failed, probably didn't wipe the faulty cache. Just restarted all jobs.

**Comment by FroMage** [comment]
So the job passed, but it's not reflected here: https://dev.azure.com/quarkus-ci/quarkus/_build/results?buildId=11841

**Comment by geoand** [comment]
Has the PR been rebased onto the latest master that fixes CI?

**Comment by FroMage** [comment]
What part of master fixed what part of CI?

**Comment by geoand** [comment]
I have no idea since I've been away the past few days.
I just saw various PRs this morning and they all had passing CI.

**Comment by aoudiamoncef** [comment]
@dmlloyd ,
All seems to be ok now :)

**Review by dmlloyd — APPROVED** [review]

**Pull request merged**